packages/client/src/views/SysAdmin/Team/user/userProfilie/UserProfile.tsx

```diff
@@ -381,33 +386,39 @@ class UserProfileComponent extends React.Component<Props, State> {
                         >
                           {user.primaryOffice &&
                             user.primaryOffice.displayLabel}
-                        </LinkButton>
+                        </LinkButtonWithoutSpacing>
                       </InformationValue>
                     </InformationHolder>
                     <InformationHolder>
-                      <InformationTitle paddingRight={208}>
-                        {intl.formatMessage(messages.roleType)}
+                      <InformationTitle>
+                        {(user.type && intl.formatMessage(messages.roleType)) ||
+                          intl.formatMessage(userFormMessages.type)}
                       </InformationTitle>
                       <InformationValue>
-                        {intl.formatMessage(userMessages[user.role as string])}{' '}
-                        /{' '}
-                        {intl.formatMessage(userMessages[user.type as string])}
+                        {(user.type &&
```

Comment:
Should this be `user.role`?  I think moving forward, we should get rid of these dynamic translation lookups altogether and opt for something like

```ts
function getUserRole(user) {
  switch(user.role) {
      Roles.FIELD_AGENT:
        return intl.formatMessage(userMessages.fieldAgent)
      Roles.REGISTRAR:
        return intl.formatMessage(userMessages.registrar)        
        default:
          unknown
  }
}
```

I think this would both help us unify the way we turn `user` properties into translated strings. Also, if I recall correctly doing dynamic lookups to messages objects is an anti-pattern as then you cannot use a tool for finding missing / deprecated translation keys.